#TestsJan202016

```diff
@@ -0,0 +1,22 @@
+#Tests on Wed, Jan 20, 2016
```

Comment:
These are nice - I'll look at setting up a proper test suite and integrating these with unittest - you're much more contentious than I am. :+1: